source/unified-test-format/tests/Makefile

```diff
@@ -53,15 +59,40 @@ retryable-reads: HAS_AJV
 retryable-writes: HAS_AJV
 	@ajv test -s $(SCHEMA) -d "../../retryable-writes/tests/unified/*.yml" --valid
 
+run-command: HAS_AJV
+	@ajv test -s $(SCHEMA) -d "../../run-command/tests/unified/*.yml" --valid
+
+server-discovery-and-monitoring: HAS_AJV
+	@ajv test -s $(SCHEMA) -d "../../server-discovery-and-monitoring/tests/unified/*.yml" --valid
+
+server-selection: server-selection/logging
+
+server-selection/logging: HAS_AJV
+	@ajv test -s $(SCHEMA) -d "../../server-selection/tests/logging/*.yml" --valid
+
 sessions: HAS_AJV
 	@ajv test -s $(SCHEMA) -d "../../sessions/tests/*.yml" --valid
 
-command-logging-and-monitoring: HAS_AJV
-	@ajv test -s $(SCHEMA) -d "../../command-logging-and-monitoring/tests/logging/*.yml" --valid
-	@ajv test -s $(SCHEMA) -d "../../command-logging-and-monitoring/tests/monitoring/*.yml" --valid
+transactions-convenient-api: HAS_AJV
+	@ajv test -s $(SCHEMA) -d "../../transactions-convenient-api/tests/unified/*.yml" --valid
 
-client-side-encryption: HAS_AJV
-	@ajv test -s $(SCHEMA) -d "../../client-side-encryption/tests/unified/*.yml" --valid
+transactions: HAS_AJV
+	@ajv test -s $(SCHEMA) -d "../../transactions/tests/unified/*.yml" --valid
+
+unified-test-format: unified-test-format/invalid unified-test-format/valid-fail unified-test-format/valid-pass
```

Comment:
The `.PHONY all` target already calls unified-test-format/invalid unified-test-format/valid-fail unified-test-format/valid-pass individually. Should we remove this?

Reply:
The `.PHONY` target lists all targets in the file so they're not checked as actual file names. The `all` target depends on the `unified-test-format` target, which in turn depends on these individual targets. This was the part that was missing in the incomplete comment and has now been clarified: for specs with multiple tests folders, we create one target for each tests folder and another target for the spec itself.